pkg/tests/archive_test.py

```diff
@@ -339,5 +339,24 @@ def testChangingRootDirectory(self):
     ]
     self.assertTarFileContent(self.tempfile, content)
 
+  def testPackageDirFileAttribute(self):
+      """
```

Comment:
Multiline docstrings should have a one line summary at the top.

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
@aiuto addressed in https://github.com/bazelbuild/rules_pkg/pull/57/commits/fa82eea821e9cadd65cc994215e8c7fdb08c5284